client/src/components/saveModal.jsx

```diff
@@ -0,0 +1,310 @@
+import styled, { css } from 'styled-components';
+import React, {Component} from 'react';
+// import {Modal} from './modalshared.jsx';
+
+const ModalContent = styled.div`
+  width: 450px;
+  position: absolute;
+  top: 10%;
+  left: 50%;
+  transform: translate(-50%, -5%);
+  background-color: white;
+  padding: 24px;
+  border-radius: 5px;
+  border-color: rgb(153, 153, 153);
+  z-index: 3000;
+`
+
+const ModalTopHeader = styled.div`
+  margin-bottom: 12px;
+  text-align: center;
+`
+const ModalTitle = styled.h2`
+  font-weight: bold;
+  font-size: 21px;
+  color: #D32323;
+  margin: 0px 0px 6px 0px;
+`
+const ModalClose = styled.span`
+  position: relative;
+  left: 50%;
+  color: #666; 
+  &:hover{
+    color: #BCBCBC;
+    cursor: pointer;
+  }
+`
+ModalClose.displayName = "ModalClose"
+
+const ShareButtonContainer = styled.div`
+  margin: 20px 0px;
+`
+const SignUpWithFacebook = styled.div`
+  text-align: center;
+  background-color: #3B5998;
+  box-shadow: 0 1px 1px rgba(0,0,0,0.3);
+  color: white;
+  width: 100%;
+  height: 100%;
+  border-radius: 5px;
+  vertical-align: middle;
+  padding: 5px 0px;
+  border-color: #263961;
+  border: 1px solid rgb(38, 57, 97);
+  font-weight: bold;
+  font-size: .9rem;
+  margin-bottom: 10px;
+  &:hover{
+    cursor: pointer;
+    background-color: #4669B5;
+  }
+`
+  
+const ShareIcon = styled.i`
+  margin-right: 15px;
+`
+
+const SignUpWithGoogle = styled.div`
+  box-shadow: 0 1px 1px rgba(0,0,0,0.3);
+  text-align: center;
+  background-color: white;
+  color: rgb(102,102,102);
+  width: 100%;
+  height: 100%;
+  border-radius: 5px;
+  vertical-align: middle;
+  padding: 5px 0px;
+  border-color: #263961;
+  border: 1px solid rgb(217, 217, 217);
+  font-weight: bold;
+  font-size: .9rem;
+  &:hover{
+    cursor: pointer;
+    background-color: #f5f5f5;;
+}
+`
+const OrFieldSet = styled.fieldset`
+  border-top: 1px solid #ccc;
+  border-bottom: 1px solid transparent;
+  text-align: center;
+  margin: 15px 0 0 0;
+  border-right: none; 
+  border-left: none; 
+  `
+const OrLegend = styled.legend`
+  font-size: .85em;
+  padding: 0 18px;
+  color: #666;
+`
+
+const SignUpForm = styled.form`
+  text-align: left;
+`
+
+const SignUpLabel = styled.label`
+  color: black;
+  font-weight: 600;
```

Comment:
I noticed you are using both bold and number for font weight, I think best practices would be to keep it consistent, and preferable using numbers (600, 700, etc)

Reply:
true - i think in my case 600 === bold because. this particular font family considers everything beyond 600 to be bold. I have made the change for consistency to "font-weight: bold"

Reply:
thanks man. I noticed it in a few places, so might be worth searching to make sure full doc is consistent